prompts/power-automate/inspection-approval-assignment/README.md

```diff
@@ -0,0 +1,32 @@
+# YOUR PROMPT NAME 
+
+Inspection Approval Assignment
+
+## Description
+
+This prompt helps with automating the process of approval and assignment of new inspections, notifying both the inspector's manager and the inspector, and updating the inspection status based on approval or rejection.
+
+## Prompt
+
+Create a flow that triggers when a new inspection is created. The flow should send an approval request to the inspector’s manager. If approved, assign the inspection to the inspector and send them a notification with the property address, inspection date, and notes. If rejected, update the inspection status to 'Rejected'. trigger should be a new item in sharepoint list
+
+### Supported Language(s)
+
+[English](./en-us/prompt.md)
+
+## Authors
+
+Solution|Author(s)
+--------|---------
+Markus Franz | (https://www.github.com/mmbr1606) 
+
+## Minimal Path to Awesome
+
+* Copy the prompt
+* Paste prompt into the tool of your choice (AIBuilder Azure Open AI, Power Apps Copilot, etc)
+
+## Disclaimer
+
+**THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.**
+
+<img src="https://m365-visitor-stats.azurewebsites.net/powerplatform-prompts/samples/ai-builder/sample" aria-hidden="true" />
```

Comment:
Please change this to `<img src="https://m365-visitor-stats.azurewebsites.net/powerplatform-prompts/prompts/power-apps/inspection-approval-assignment" aria-hidden="true" />`